### DISCUSSION TITLE
How to set the transformer model directory in an experimental project?

### DISCUSSION THREAD

**Discussion by Bachstelze** [explosion/spaCy#10892] [Help: Coding & Implementations] [answered]
When we run `spacy project run train-transformer" the defined transformer model is loaded from huggingface.
How can i set the directory path, if i already have the model on the disk?

**Comment by polm** [comment, marked as answer]
What project are you using? The `train-transformer` command is not a built-in, it's defined in the project file.

For how to specify HuggingFace models in general see #10768. You can just put the path in the `name` field.

**Reply by Bachstelze** [reply]
I am using the experimental ud_benchmark project. I didn't know that a path is also a valid name.
Thanks for your fast response!
